examples/js/components/FileInput.jsx

```diff
@@ -4,7 +4,11 @@ import {FileInput} from 'safe-framework'
 
 class Area extends Component {
   onChange (value) {
-    window.alert(value)
+    var alertText = ' '
```

Comment:
The example might be cooler if we actually read in the file and alert the text of the file:

```javascript
onChange (file) {
  const reader = new window.FileReader()

  reader.onload = function () {
    window.alert(reader.result)
  }
  reader.readAsText(file, 'utf-8')
}
```

Also, you should always use const/let instead of var since we are writing ES6.